lesson_12/io/io_app/src/test/java/com/codedifferently/lesson12/factory/jamirabailey/CsvDateLoader.java

```diff
@@ -0,0 +1,61 @@
+package com.codedifferently.lesson12.factory.jamirabailey;
+
+import com.codedifferently.lesson12.factory.LibraryCsvDataLoader;
+import com.codedifferently.lesson12.models.CheckoutModel;
+import com.codedifferently.lesson12.models.LibraryDataModel;
+import com.codedifferently.lesson12.models.LibraryGuestModel;
+import com.codedifferently.lesson12.models.MediaItemModel;
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Service
+public class CsvDataLoader implements LibraryCsvDataLoader {
+
+    @Override
+    public LibraryDataModel loadData() {
```

Comment:
Looks like you're making progress, so feel free to submit an updated version if you'd like more credit for this assignment.